### PR TITLE
Issue #228 fix the link in hello.html

### PR DESCRIPTION
**Pull request by afarber**
- Fix the link in hello.html
- Fix typo: posing -> posting
- Remove obsolete link to websocket-jetty-api
- Update servlet 5 example to servlet 6

### CODE REVIEW

File: webapps/hello-servlet-6/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <artifactId>jetty-webapp-examples</artifactId>
     <version>12.0.x</version>
   </parent>
-  <artifactId>hello-servlet-5</artifactId>
```
Comment (joakime) [inline webapps/hello-servlet-6/pom.xml]: Please do not change the `hello-servlet-5`.

I have no problem with introducing a new `hello-servlet-6` tho.

**Review by joakime — CHANGES_REQUESTED** [review]
Please do not change `hello-servlet-5`, leave that alone.
If you want a `hello-servlet-6`, then go ahead and make that a new module in webapps.

File: webapps/hello-servlet-6/pom.xml (hunk 1 above)
Comment (afarber) [reply webapps/hello-servlet-6/pom.xml]: I have reverted servlet 6 as you have suggested

**Review by joakime — APPROVED** [review]

**Comment by joakime** [comment]
@afarber merged!  thank you for the PR.

**Pull request merged**